src/components/Navbar.jsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-trailing-spaces */
```

Comment:
- Kindly, you disabled some functionality of the linter, :warning: the linter Lint checks against best practices, code style guideline violations, and structural problems. so please delete line number 1-3 :+1: